src/EditorFeatures/Test/Preview/PreviewWorkspaceTests.cs

```diff
@@ -238,6 +238,19 @@ public void TestPreviewWorkspaceDoesNotLeakSolution()
             solutionObjectReference.AssertReleased();
         }
 
+        [Fact]
```

Comment:
Consider adding the work item attribute to link to the bug here so if anybody wonders why we're asserting this it's clear it was for memory leak reasons. (I can imagine somebody going "oh, this is a stupid test, people should call Dispose()" and we're back here again)

Reply:
Will amend the last comment to add comments and work item links.